packages/workflow/src/Interfaces.ts

```diff
@@ -593,7 +598,11 @@ export interface IExecuteSingleFunctions {
 	getItemIndex(): number;
 	getMode(): WorkflowExecuteMode;
 	getNode(): INode;
-	getNodeParameter(parameterName: string, fallbackValue?: any): NodeParameterValueType | object;
+	getNodeParameter(
+		parameterName: string,
+		fallbackValue?: any,
+		options?: IGetNodeParameterOptions,
+	): NodeParameterValueType | object;
```

Comment:
This change had to be made six times. At a glance these interfaces are all very similar to each other. Is there anything against consolidating them to remove so much repetition? Perhaps as tech debt.

Reply:
I think it'd be worth doing. I'm also pretty sure there's functions for creating each type as well that could probably be consolidated.